packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -216,10 +218,11 @@ function makeAugmentedSchema(
         });
 
         const otherDirectives = (definition.directives || []).filter(
-            (x) => !["auth", "exclude"].includes(x.name.value)
+            (x) => !["auth", "exclude", "node"].includes(x.name.value)
```

Comment:
Maybe we should move this to a constant or define a more explicit way of registering directives?